COLLABORATOR_GUIDE.md

```diff
@@ -0,0 +1,96 @@
+# jade Collaborator Guide
+
+**Contents**
+
+* Issues and Pull Requests
+* Accepting Modifications
+ - Involving the TC
+* Landing Pull Requests
+ - Technical HOWTO
+
+This document contains information for Collaborators of the jade
+project regarding maintaining the code, documentation and issues.
+
+Collaborators should be familiar with the guidelines for new
+contributors in [CONTRIBUTING.md](./CONTRIBUTING.md).
+
+## Issues and Pull Requests
+
+Courtesy should always be shown to individuals submitting issues and
+pull requests to the jade project.
+
+Collaborators should feel free to take full responsibility for
+managing issues and pull requests they feel qualified to handle, as
+long as this is done while being mindful of these guidelines and the
+opinions of other Collaborators.
+
+Collaborators may **close** any issue or pull request they believe is
+not relevant for the future of the jade project. Where this is
+unclear, the issue should be left open for several days to allow for
+additional discussion. Where this does not yield input from jade
+Collaborators or additional evidence that the issue has relevance, the
+issue may be closed. Remember that issues can always be re-opened if
+necessary.
+
+## Accepting Modifications
```

Comment:
A much cleaner commit log that reflects the order in which things were actually added to master, rather than the order in which they were written.  I wish GitHub had a "Big Green Rebase In" button (maybe one day I'll build a browser extension to add one).